src/test/resources/mmri/2b/stops.txt

```diff
@@ -1,3 +1,3 @@
-stop_id,stop_name,stop_lat,stop_lon

```

Comment:
Out of interest: are you using this data in a test?

Reply:
Yes, there exists an GTFS test for wheelchairs, this will fail if the stops are not available for wheelchair. When this test was created it was sufficient that the trip was accessible for wheelchair.